dvc/command/plots.py

```diff
@@ -151,6 +156,13 @@ def add_parser(subparsers, parent_parser):
         help="Plots file to visualize. Shows all plots by default.",
         metavar="<path>",
     ).complete = completion.FILE
+    plots_diff_parser.add_argument(
+        "-e",
+        "--experiment",
```

Comment:
No strong opinion, but consider just `--exp` instead of both. `-e` might come in handy later for the environment or something.